### DISCUSSION TITLE
add back playwright

### DISCUSSION THREAD

**Discussion by HellishBro** [llm-workflow-engine/llm-workflow-engine#319] [General]
add back playwright or at least make it not-deprecated. i think it is way easier to set up a playwright session, and I do not care about the speed.

**Comment by thehunmonkgroup** [comment]
> add back playwright

Playwright is still present, and is the mechanism used by the browser backend.

> or at least make it not-deprecated

The developers deprecated the browser backend for valid reasons after careful consideration, it's unlikely this decision will change.

> i think it is way easier to set up a playwright session

Configuring the browser backend is trivial, adding one value to a configuration file, as documented here: https://github.com/mmabrouk/chatgpt-wrapper#playwright-browser-based-deprecated
